townhall/myapi/views.py

```diff
@@ -25,6 +26,14 @@ def handle_volunteer_request(self, request):
         
         serializer = VolunteerSerializer(volunteer_obj)
         return Response(serializer.data, status=status.HTTP_200_OK)
+    
+    # DELETE Volunteer
+    @action(detail=False, methods=['delete'], url_path='volunteer')
+    def handle_volunteer_delete(self, request):
+        volunteer_id = self.request.query_params.get('id')
```

Comment:
minor comment: if `id` is not given in query param, then throw error , otherwise this will not graceously error out